release-notes/v1.3.0.md

```diff
@@ -18,3 +18,5 @@ The following table lists the known issues for HPE CSI Driver for Kubernetes v1.
 |--|---------|-----|-----------|----------|
 |CON-1107|csi.k8s|Restoring NFS access on extended outage with backends| If PVCs are created with NFS access using `HPE CSI Driver`, then a `NFS Server Provisioner` instance will be created for each PVC. Since these servers are backed by underlying FC/iSCSI disks, if there is an extended outage (> 150s) with the backend, then NFS servers will get I/O failures and cannot service NFS shares anymore. Since all filehandles are stale, NFS servers do not automatically recover if access is restored later on and application pods can be hung due to hard mounts|In these cases we recommend to restart affected NFS Server Provisioner pods or the node to restore iSCSI/FC disk access again and serve NFS mounts|
 |CON-1122|csi.k8s|iSCSI CHAP credentials applied are not reflected using iscsiadm command|When iSCSI CHAP settings are modified with `HPE CSI Driver`, credentials are updated using iscsiadm command when next pod is scheduled onto the node. However, these credentials are not displayed using iscsiadm command with already existing sessions|This is just cosmetic and current CHAP settings are updated with each iSCSI node entry under /etc/iscsi/nodes or /var/lib/iscsi/nodes directories. When iSCSI login is attempted again, latest values will be used by iscsid|
+|CON-1260|csi.k8s|iSCSI CHAP username/password is not updated in HPENodeInfos custom resource|HPE CSI driver maintains iSCSI CHAP credentials within each HPENodeInfos custom resource instance. When driver is re-deployed with changes to iSCSI CHAP username or password, sometimes HPENodeInfos instances are not updated/re-created causing them to contain stale data. This causes failures when application pod is scheduled onto the node with stale iSCSI CHAP username/password|Possible workarounds for this include:<br />1. Delete HPENodeInfos instances manually and re-deploy CSI driver to create them with updated values.<br />2. Edit HPENodeInfos to update iSCSI CHAP username/password manually with each instance. Password has to be base64 encoded value|
+|CON-1266|csi.k8s|Failure to edit PVC size and other volume properties at same time|With introduction of HPE Volume Mutator from v1.3.0, users can now edit support volume parameters through PVC annotations. But, when volume properties are edited along with size at the same time, both CSI Resizer and HPE Volume Mutator might attempt PVC updates(i.e status and conditions). This might cause errors with either one of them during update saying resource version has changed externally and update process might fail|Avoid editing volume properties through annotations along with size at the same time. Resize is a different operation which involves controller/node expansion triggered through CSI external resizer. Its recommended to treat this as separate edit than other backend volume properties|
```

Comment:
at **the** same time...
With **the** introduction of **the** HPE ...
users can now edit volume parameters... <= drop support
both **the** CSI Resizer and **the** HPE Volume Mutator might...
Its recommended... <= It is recommended
Its recommended to treat this as separate edit than other backend volume properties <= I don't understand this sentence

Reply:
Agreed, updated the changes..

Also, we can rephrase the last statement as follows:
`It is recommended to perform resize operation (volume size edit) separately and editing the rest of the volume properties like limitIops, limitMbps, etc.. as a different operation.`